controller/OrderController.go

```diff
@@ -0,0 +1,133 @@
+package controller
+
+import (
+	"fmt"
+	"github.com/gin-gonic/gin"
+	"net/http"
+	"stlab.itechart-group.com/go/food_delivery/courier_service/dao"
+	"strconv"
+)
+
+type listOrders struct {
+	Data []dao.DetailedOrder `json:"data"`
+}
+
+// @Summary GetCourierCompletedOrders
+// @Description get list of completed orders by courier id
+// @Tags order
+// @Produce json
+// @Param page query int true "page"
+// @Param limit query int true "limit"
+// @Param idcourier query int true "idcourier"
+// @Success 200 {object} listOrders
+// @Failure 400 {string} string
+// @Failure 500 {string} string
+// @Router /orders/completed [get]
+func (h *Handler) GetCourierCompletedOrders(ctx *gin.Context) {
+	page, er := strconv.Atoi(ctx.Query("page"))
+	if er != nil || page == 0 {
+		ctx.JSON(http.StatusBadRequest, gin.H{"message": "expect an integer greater than 0"})
+		return
+	}
+	limit, er1 := strconv.Atoi(ctx.Query("limit"))
+	if er1 != nil || limit == 0 {
+		ctx.JSON(http.StatusBadRequest, gin.H{"message": "expect an integer greater than 0"})
+		return
+	}
+	idCourier, er := strconv.Atoi(ctx.Query("idcourier"))
+	if er != nil {
+		ctx.JSON(http.StatusBadRequest, gin.H{"message": "expect an integer greater than 0"})
```

Comment:
🔴 Error messages should be more informative. The user should rapidly understand what exactly field caused the error.

`expect an integer greater than 0` -> `"page" query param is wrong. Expected an integer greater than 0`
`expect an integer greater than 0` -> `"limit" query param is wrong. Expected an integer greater than 0`
etc.